frontend/src/components/MyProfile.vue

```diff
@@ -156,6 +216,10 @@
         private message: string = "";
         private userData!: UserProfileInfoResponse;
         private avatar: string = '/img/no-avatar.png';
+        private showEditName: boolean = false;
+        private showEditUsername: boolean = false;
+        private name: string = "";
+        private username: string = "";
```

Comment:
maybe rename these one (name, username)  to something like editableName, editableUsername?